src/applications/vaos/tests/services/appointment/transformers.unit.spec.js

```diff
@@ -342,10 +342,7 @@ describe('VAOS Appointment transformer', () => {
       });
 
```

Comment:
Removing test due to the following change https://github.com/department-of-veterans-affairs/vets-website/pull/16739#discussion_r615030943